settings-schema/settings.json

```diff
@@ -12,7 +12,7 @@
             "type": "boolean",
             "title": "Show Deploy Notebook button",
             "description": "If set, a button will be shown in the toolbar to deploy the notebook to Ploomber Cloud.",
-            "default": true
+            "default": false
```

Comment:
This only hides the button by default, right? A user can still enable it by changing the setting on their own.

I think to avoid confusion, we should remove the option (and all of its logic) altogether

Reply:
I didn't find the `enable share notebook button` setting. I'm not sure if I'm missing something here, but users are going to install the package with the settings as it is.  How will they access this settings file? 

But anyways I'll remove the code altogether !

Reply:
@neelasha23 In Jupyter Lab, users can access the settings through `Settings` -> `Settings Editor` -> `JupySQL`. From there they can toggle the `Format SQL` button and the `Share Notebook` button. Here's a screenshot for reference:

![image](https://github.com/ploomber/jupysql-plugin/assets/42661667/3f068453-05c1-49d4-8a93-cae93e67abe8)

But yes I think it would be best to remove it altogether!

Reply:
Removed the files, please check again @bryannho